<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Protein Folding</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="verticalWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <widget class="QPushButton" name="stopButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>151</width>
            <height>26</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;background-color:#DC143C;font-size:15px</string>
          </property>
          <property name="text">
           <string>Стоп</string>
          </property>
         </widget>
         <widget class="QPushButton" name="startButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>151</width>
            <height>26</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;background-color:green;font-size:15px</string>
          </property>
          <property name="text">
           <string>Старт</string>
          </property>
         </widget>
         <widget class="QPushButton" name="settingsButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>151</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;background-color:#00BFFF;font-size:12px</string>
          </property>
          <property name="text">
           <string>Настройки</string>
          </property>
         </widget>
         <widget class="QLabel" name="countConvolution">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>210</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>count</string>
          </property>
         </widget>
         <widget class="QLabel" name="error">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>161</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>error</string>
          </property>
         </widget>
         <widget class="QLabel" name="rating">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>230</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>rating</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="bestTime">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>270</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>time</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="onlyBetter">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>161</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Показать только лучший</string>
          </property>
         </widget>
         <widget class="QLabel" name="avarageRating">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>250</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>averageRaiting</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="showConnection">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>161</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Показать связи</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="showGraph">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>161</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Показывать результаты</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
